doc/rados/command/list-inconsistent-obj.json

```diff
@@ -0,0 +1,140 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "epoch": {
+      "description": "Scrub epoch",
+      "type": "integer"
+    },
+    "inconsistents": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "object": {
+            "description": "Identify a Ceph object",
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string"
+              },
+              "nspace": {
+                "type": "string"
+              },
+              "locator": {
+                "type": "string"
+              },
+              "snap": {
+                "oneOf": [
+                  {
+                    "type": "string",
+                    "enum": [ "head", "snapdir" ]
+                  },
+                  {
+                    "type": "integer",
+                    "minimum": 0
+                  }
+                ]
+              }
+            },
+            "required": [
+              "name",
+              "nspace",
+              "locator",
+              "snap"
+            ]
+          },
+          "errors": {
+            "description": "Union of all shard errors, including errors in which the faulty shard is NOT identified",
+            "type": "array",
+            "items": {
+              "enum": [
+                "attr_unexpected",
+                "missing",
+                "stat_error",
+                "read_error",
+                "data_digest_mismatch",
+                "omap_digest_mismatch",
```

Comment:
@athanatos 

data_digest_mismatch: Shard computed digest doesn't match the **selected** shard
data_digest_mismatch_oi: Shard computed digest doesn't match its own object info digest
ec_hash_error: Shard computed digest doesn't match EC recorded hash

I'm think that the following errors should have a prefix because they are the errors having to do with comparison to selected (authoritative) shard.  Maybe we should add "compare_" or something else?  Also, they are the ones which will be documented as less significant if **comparing_random_osd** is set.

data_digest_mismatch
omap_digest_mismatch
size_mismatch
attr_value_mismatch
attr_key_mismatch